packages/client/hmi-client/src/types/Types.ts

```diff
@@ -463,6 +464,20 @@ export interface Annotations {
     model_types?: string[];
 }
 
+export interface Card {
+    DESCRIPTION?: string;
```

Comment:
These are captials I believe because of jsonProperty

Reply:
Yeah, they come in as capitals 
https://github.com/DARPA-ASKEM/knowledge-middleware/blob/main/tests/scenarios/basic/model_card.json

Reply:
This is the kind of things we don't have time to clean, but we should...
Maybe we will have some magic time to do so.

Reply:
These are capitals though because the wrong annotation is being used - im not sure if we're going to run into serialization issues on the back end because of it